python-frontend/src/test/java/org/sonar/python/semantic/v2/typeshed/TypeShedDescriptorsProviderTest.java

```diff
@@ -41,14 +39,18 @@ class TypeShedDescriptorsProviderTest {
   @RegisterExtension
   public LogTesterJUnit5 logTester = new LogTesterJUnit5().setLevel(Level.DEBUG);
 
-  @BeforeEach
-  void setup() {
-    ProjectPythonVersion.setCurrentVersions(PythonVersionUtils.allVersions());
+  private static TypeShedDescriptorsProvider cleanProvider() {
```

Comment:
_Minor:_ I think this could just be called `typeshedProvider` for clarity.
It took me a minute trying to understand what `var provider = cleanProvider();` meant without looking at this method (note how the `var` adds to the confusion here 😅 )